### ISSUE TITLE
Nextcloud stuck at "Update in process"

### ISSUE THREAD

**Issue by bheberling** [nextcloud/docker#1909]
While updating Nextcloud to 25.0.3 the process halted at either "Move new files in place" or "Delete old files". Now during web access I only receive the message "Update in process."
I tried logging into the docker container and running updater.phar and received the following error at the Downloading step: 

`Downloading ...PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted (tried to allocate 134217736 bytes) in phar:///config/www/nextcloud/updater/updater.phar/lib/Updater.php on line 781`

Per a forum post I added an environment variable to the docker container: PHP_MEMORY_LIMIT=512M
Now, when I run the updater I receive another error: Step 4 is currently in process. Please call this command later.

Any suggestions? Thanks for any help you can provide.

**Comment by joshtrichards** [comment]
Hi @bheberling - Sorry to hear you ran into some difficulties. Can you share your Docker compose file as well as the precise rocess you utilized to trigger the initial update? This will be helpful to determine how to reproduce the issue as well as whether or not it's a configuration matter or what. Thanks!

**Comment by joshtrichards** [comment]
Also sounds potentially somewhat similar to #1904.

**Comment by bheberling** [comment]
@joshtrichards, thanks for the response - and my apologies, I didn't realize I had left this issue open. I ended up deleting the container and setting up Nextcloud again from scratch, and haven't had an issue since. Unfortunately I can't remember any details of how I ran into the problem apart from what I've shared above, which makes it difficult to debug - in the future I'll document my issues better. I'll go ahead and close this.
